src/bootstrap/config.toml.example

```diff
@@ -129,6 +129,7 @@
 # with debuginfo (the -g flag)
 #optimize-tests = true
 #debuginfo-tests = true
+#codegen-tests = true
```

Comment:
This is actually slightly different from the above config because it *disables* tests rather than configure how they're compiled. Perhaps this could have its own doc block explaining it?